csv/result_internal_test.go

```diff
@@ -0,0 +1,7 @@
+package csv
+
+import "sync/atomic"
+
+func (d *tableDecoder) IsDone() bool {
```

Comment:
Yes. Basically, if it has a `_test.go` suffix it will only be compiled in with tests. But you can mix test code into the original package and also have test code be part of a separate package.